torch/distributed/rpc/api.py

```diff
@@ -64,24 +65,38 @@ def wait_all_workers():
         >>> result = rpc.rpc_sync("worker1", torch.add, args=(torch.ones(1), 1))
         >>> # ready to shutdown
         >>> rpc.wait_all_workers()
+        >>> rpc.shutdown()
 
         On worker 1:
         >>> import torch.distributed.rpc as rpc
         >>> rpc.init_rpc("worker1", rank=1, world_size=2)
         >>> # wait for worker 0 to finish work, and then shutdown.
         >>> rpc.wait_all_workers()
+        >>> rpc.shutdown()
     """
     global _agent
```

Comment:
(This can come in a followup PR)

We might not need this `global _agent` any more as we are not modifying it.